components/VendorCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import ButtonLink from '../components/ButtonLink';
+import { Vendor } from "typings";
+
+interface VendorCardProps {
+    vendor: Vendor;
+}
+
+function VendorCard(props: VendorCardProps): JSX.Element {
+    const { vendor } = props;
+    const {
+        name,
+        logoUrl,
+        url
+    } = vendor;
+
+    return (
+        <div className="vendor-card">
+            <div className="image">
+                <img src={logoUrl === undefined || logoUrl === '' ? '/images/empty-base-kit-illu.svg' : logoUrl} alt="Vendor Logo" />
```

Comment:
base kit illu is niks bij vendors. Die kan je for now even weg halen, moet een andere illu komen.

Reply:
Jaa ik weet, gewoon string leeg laten?